### PR TITLE
[Utils] Fix syntax error in gitian-build.sh

### PR DESCRIPTION
**Pull request by aigon1989**
The gitian-build.sh script was broken when I've tried to use it to compile my own wallet so I've fixed it.

### CODE REVIEW

File: contrib/gitian-build.sh [hunk 1]
```diff
@@ -314,7 +314,8 @@ then
 	    ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
 	    ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-aarch64 --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
 	    mv build/out/pivx-*.tar.gz build/out/src/pivx-*.tar.gz ../pivx-binaries/${VERSION}
-	popd
+	    popd
+	fi
```
Comment (Fuzzbawls) [inline contrib/gitian-build.sh]: `fi` should come before `popd`

Comment (aigon1989) [reply contrib/gitian-build.sh]: Thank you pointing out my mistake using spaces as indents, I've edited it on github editor without realizing it, and sorry for the inconvenience.

Thank you

**Comment by Fuzzbawls** [comment]
@aigon1989 are you going to fix/amend this?

**Comment by aigon1989** [comment]
Sorry, I thought that you have fixed it. It is my first pull request contributing to a github project.

If I didn't understood it wrong, it is needed to replace tabs by spaces, right?

I can do a whole file replace if needed. 

Also, I've found that the "commitFiles" if statement is in the wrong place because it is place inside the "build" if statement, and it looks like it is a whole distinct script step. Can you confirm that? Or another pull request should be made for that issue?

Thank you again,

Regards,

Aitor

**Comment by Fuzzbawls** [comment]
Changed/New lines should be indented with spaces, not tabs (despite however the file was initialized).

Both of the "commitFiles" checks are in their proper place, and do not need editing.

File: contrib/gitian-build.sh [hunk 2]
```diff
@@ -260,153 +260,155 @@ popd
...
+    # AArch64
+    if [[ $aarch64 = true ]]
+    then
+        echo ""
+        echo "Compiling ${VERSION} AArch64"
+        echo ""
+        ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
+        ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-aarch64 --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
+        mv build/out/pivx-*.tar.gz build/out/src/pivx-*.tar.gz ../pivx-binaries/${VERSION}
+        #popd
```
Comment (Warrows) [inline contrib/gitian-build.sh]: Nit: I don't think this comment needs to stay here

Comment (aigon1989) [reply contrib/gitian-build.sh]: I have ammended the comment and arranged the commit history  as told.

Thank you

**Pull request merged**